contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -134,10 +134,12 @@ version(PG_FUNCTION_ARGS)
 {
 	StringInfoData temp;
 	void *info;
+	const char *sql_server_version;
 
 	initStringInfo(&temp);
 
-	if (pg_strcasecmp(pltsql_version, "default") == 0)
+	sql_server_version= GetConfigOption("babelfishpg_tsql.version", true, false);
```

Comment:
null check

Reply:
PG will throw an error(``ERROR: zero-length delimited identifier at or near """"``) when user trying to set ``babelfishpg_tsql.version`` to null